specutils/tests/test_spectrum1d.py

```diff
@@ -110,3 +111,18 @@ def test_create_with_uncertainty():
 
     assert spec.flux.unit == spec.uncertainty.unit
 
+
+def test_read_linear_solution():
+    file_path = os.path.join(os.path.dirname(__file__),
+                             'data/L5g_0355+11_Cruz09.fits')
+
```

Comment:
@nmearl @brechmos-stsci  Is this the best way to be reading in this example file ?

Reply:
it can be done using the same machinery as in the other file? `specutils/tests/test_loaders.py`, though probably the test can be even merged.

Reply:
The test was already merged on #184 Clearly I messed up the branches. This was my very first PR I think.